src/Components/Navbar.js

```diff
@@ -1,37 +1,37 @@
-import React  from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import React from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Tab from '@material-ui/core/Tab';
 import TabContext from '@material-ui/lab/TabContext';
 import TabList from '@material-ui/lab/TabList';
+import styled from 'styled-components';
 
-const useStyles = makeStyles((theme) => ({
-    root: {
-        flexGrow: 1,
-        backgroundColor: theme.palette.background.paper,
-    },
-}));
+const StyledAppBar = styled(AppBar)`
+        background : #222;
+    `;
+
+const Div = styled('div')`
+        flex-grow: 1;
+`
```

Comment:
I would suggest  change this to be like this:
```
const TabContextStyled = styled(TabContext)`
  flex-grow: 1;
`;

export default function NavBar() {

    const [value, setValue] = React.useState('1');

    const handleChange = (event, newValue) => {
        setValue(newValue);
    };

    return (
        <TabContextStyled value={value}>
            <StyledAppBar position="static">
                <TabList onChange={handleChange}>
                    <Tab label="Users" value="1"/>
                    <Tab label="Products" value="2"/>
                    <Tab label="Manufacturers" value="3"/>
                </TabList>
            </StyledAppBar>
        </TabContextStyled>
    );
}
```

Because when we add the `flex-grow` style to the `TabContent` we wouldn't need that extra wrapping `div`.
Also I think adding `...Styled` to  end is better that adding it to the beginning.